manifests/configmap.yaml

```diff
@@ -32,7 +32,7 @@ data:
   team_admin_role: "admin"
   teams_api_url: http://fake-teams-api.default.svc.cluster.local
   workers: "4"
-  enable_load_balancer: "true"
+  enable_load_balancer: "false"
```

Comment:
That somehow prevents allocation of external IPs for the operator service on `minikube`. Are there other settings that need to be changed ?